test/rules/no-default-export/test.ts.lint

```diff
@@ -0,0 +1,4 @@
+export const default = 'VALID';
+
+export default 'INVALID';
+       ~~~~~~~ [use of default exports is disallowed]
```

Comment:
I really didn't know what else to write down as a test, but tbh i didn't spend much time thinking about it. I'll add the above in there for good measure.